### DISCUSSION TITLE
Cannot remember password

### DISCUSSION THREAD

**Discussion by khoaminhbui** [halo/macosvpn#55] [Q&A]
Firstly, thank you for creating this app. 

I forked the repo and ran well. It can create the Cisco IPSec VPN with 2 records on the Keychain (Shared secret & Auth Password).
But it's strange is that at the first attempt to connect, a popup appeared and asked for password. After entering password it could connect successfully, and (here it came) the Auth Password record in Keychain automatically deleted.

Anyone has an idea about this weird thing?

**Comment by khoaminhbui** [comment]
Updated: The OS is Monterey on a M1 chip.

**Comment by halo** [comment]
Hi! How did you run it? From within Xcode with the "play"-button, or did you compile and then run it in the Terminal with sudo?

**Reply by khoaminhbui** [reply]
I ran both ways: from Terminal with the executable downloaded from this repo and from XCode with `play` button.
